jest.config.js

```diff
@@ -8,6 +10,11 @@ const supportsESMAndJestLightRunner = semver.satisfies(
 );
 const isPublishBundle = process.env.IS_PUBLISH;
 
+if (!supportsESMAndJestLightRunner) {
+  //Avoid source maps from breaking stack tests.
+  shell.rm("-rf", "packages/babel-core/lib/**/*.js.map");
+}
```

Comment:
I don't like that this deletes source maps just because I need to run the tests, but it's good enough for now.

Reply:
Yes, I hate that too. But I've tried a few things and this seems to work best.😓